livekit-ffi/src/server/room.rs

```diff
@@ -465,6 +465,28 @@ impl RoomInner {
         server.watch_panic(handle);
         proto::UpdateLocalNameResponse { async_id }
     }
+
+    pub fn update_local_attributes(
+        self: &Arc<Self>,
+        server: &'static FfiServer,
+        update_local_attributes: proto::UpdateLocalAttributesRequest,
+    ) -> proto::UpdateLocalAttributesResponse {
+        let async_id = server.next_id();
+        let inner = self.clone();
+        let handle = server.async_runtime.spawn(async move {
+            let _ = inner
```

Comment:
We should not ignore the error

Reply:
I see the errors are also not exposed for update_metadata, can you add it inside this PR?